include/deal.II/matrix_free/cuda_matrix_free.templates.h

```diff
@@ -274,7 +274,9 @@ namespace CUDAWrappers
       cudaError_t error_code = cudaMemcpyToSymbol(
         constraint_weights,
         shape_info.data.front().subface_interpolation_matrix.data(),
-        sizeof(double) * fe.n_dofs_per_face(0) * fe.n_dofs_per_face(0));
+        sizeof(double) *
+          shape_info.data.front().subface_interpolation_matrix.size());
```

Comment:
Thanks! But shouldn't that be the same?

Reply:
That's what I thought too but it's not. It is the same in 2D but not in 3D.

Reply:
Do you happen to have an example of values? I'll take a look at that later. Independent of this, this change is fine to merge!

Reply:
Right, these are 1D interpolation matrices. Makes all sense! Thanks for the fix!

Reply:
It's certainly more consistent this way to get the pointer and the number of elements from the same place.